apps/src/templates/instructions/inline-audio.module.scss

```diff
@@ -1,5 +1,42 @@
 @import "../../mixins.scss";
+@import "color.scss";
 
 .inlineAudioButton {
   @include remove-button-styles(inherit, inherit, inherit);
+  margin-left: 3px;
+  margin-right: 3px;
+  display: flex;
+
+  .iconWrapper {
+    background-color: $lightest_purple;
+    cursor: pointer;
+    float: left;
+    border: none;
+    outline: none;
+    width: 33px;
+    box-sizing: border-box;
+
+    &Volume {
+      border-radius: 4px 0 0 4px;
+    }
+
+    &PlayPause {
+      border-radius: 0 4px 4px 0;
+    }
+
+    .buttonImg {
+      opacity: 1;
+      float: left;
+      padding-right: 8px;
+      padding-left: 8px;
+      color: $dark_charcoal;
+    }
+  }
+
+  &:hover,
```

Comment:
can you put this inside of `.iconWrapper` instead of having it separate here?

Reply:
Yeah so this is a little weird because the desired outcome is that when you hover over the button, both icons (volume and play pause) change background color. If this was in `.iconWrapper`, then only the icon you were hovering over would change.

Reply:
makes sense!